Bottom: 6aaa3ee1a6f0095f976f12a7c3890e1da600a80d
Top:    f51604fb6a6785025e54c2581d30045f95fe626f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-04-21 20:54:31 +0100

Fix rubberbanding not to touch rat-lines


---

diff --git a/src/rubberband.c b/src/rubberband.c
index aec6913..e54d61a 100644
--- a/src/rubberband.c
+++ b/src/rubberband.c
@@ -78,7 +78,7 @@ test_circle_hits_box (PointType *point, Coord radius, BoxType *box)
       box->Y1 - radius <= point->Y && point->Y <= box->Y2 + radius)
     {
       /* We are close enough to the box boundary that we might touch
-       * it, althoguh it is possible that if we are near one of the
+       * it, although it is possible that if we are near one of the
        * box corners, the circle contour doesn't touch the box.
        */
